Bendy and the Dark Revival/bendyandthedarkrevival.asl

```diff
@@ -109,8 +109,22 @@ onStart
 update
 {
     current.IsLoadingSection = vars.Helper.Read<IntPtr>(current.gm + 0xD0) != IntPtr.Zero;
-    current.IsPaused = current.PauseMenuActive && current.GameState == 4 && current.GMIsPaused && current.IsPauseReady;
-    current.IsLoading = current.IsLoadingSection || (settings["remove_paused"] && current.IsPaused);
+    current.IsLoading = current.IsLoadingSection;
+
+    if(current.playerState != old.playerState)
```

Comment:
probably shouldn't be committing these debug statements? unless you intentionally want them to be shipped out, if so that's probably fine